user/getting-started.md

```diff
@@ -1,51 +1,84 @@
 
 # Getting Started
 
-Right now (2017Q2) the easiest way to get started is to use local driver development environment: manually clone and build a container for a language driver with an SDK, then run it through Docker in order to get source -> UAST.
-
-i.e for a [Python driver](https://github.com/bblfsh/python-driver) :
+First thing you need to use Babelfish is running the Babelfish Server.
 
 ## Prerequisites
- - Docker
- - Go, for SKD build
- - (macOS) `coreutils` and `gettext`, for SDK to work
 
-## Build
+### Linux
 
-**Build SDK**
-```
-go get -u github.com/bblfsh/sdk/...
+- [Docker](https://www.docker.com/community-edition) (optional)
+
+### macOS
+
+- [Docker for Mac](https://www.docker.com/docker-mac)
+
+### Windows
+
+- [Docker for Windows](https://www.docker.com/docker-windows)
+
+## Running with Docker (recommended)
+
+The easiest way to run the server is using Docker. You can start it with the
+following command:
+
+```bash
+$ docker run --privileged -p 9432:9432 --name bblfsh bblfsh/server
 ```
 
-**Build driver + container**
+If everything worked, it should output something like this:
+
 ```
-# Ignore the 'no buildable Go source files' message
-go get -u github.com/bblfsh/python-driver
+time="2017-06-01T09:12:22Z" level=debug msg="binding to 0.0.0.0:9432" 
+time="2017-06-01T09:12:22Z" level=debug msg="initializing runtime at /tmp/bblfsh-runtime" 
+time="2017-06-01T09:12:22Z" level=debug msg="starting server" 
+time="2017-06-01T09:12:22Z" level=debug msg="registering gRPC service" 
+time="2017-06-01T09:12:22Z" level=info msg="starting gRPC server" 
+```
+
+The only mandatory flag is [`--privileged`](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities).
+Babelfish Server uses containers itself to run language drivers, which currently
+requires it to run in privileged mode.
+
+Exposing the Babelfish port (`9432`) with `-p 9432:9432:` makes it easier to
+connect to it from outside the container.
 
-cd $GOPATH/src/github.com/bblfsh/python-driver
-go get -v -t ./...
-bblfsh-sdk prepare-build
-make build
+Now you can test that it works by submitting a file for parsing:
+ 
 ```
+$ echo "import foo" > sample.py
+$ docker run -v $(pwd):/work --link bblfsh bblfsh/server bblfsh client --address=bblfsh:9432 /work/sample.py
+```
+
+First request might timeout, since the server has to fetch the required driver
+before responding. If it does, just retry.
```

Comment:
During the process the server will log what it's doing. It may be useful to note this here, since it makes it easier to check that the delay is due to getting drivers.